CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 - Added a button to fit the viewport and timebar to vessel tracks
 - Removed automatic fit to viewport and timebar when clicking on a vessel search result (can now be done by explicitely clicking the button)
 - Clicking the (i) vessel button again now hides vessel details
-- Harmonisation of icons
+- Harmonisation of icons and various samll UI fixes
+- Right panel automatically expands when clicking on a vessel on the map
```

Comment:
@nerik another suggestion for the changelog is that we should start using the same format we have in other projects:
`pullrequest-link - Description`, that way a user interested in knowing how a given change was done can easily navigate to the PR. You can check the PIPE-[x] pr repos to see that structure.